src/Components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,129 @@
+$grey-text-1-100: rgba(101, 108, 115, 1);
+$grey-text-1-70: rgba(101, 108, 115, 0.7);
```

Comment:
Add these colour variables in `index.scss` inside `:root` and them use it using `var(variable-name)`

Reply:
Should I add this in the home-page branch as well separately?